.coderabbit.yml

```diff
@@ -0,0 +1,40 @@
+language: "en"
+early_access: false
+reviews:
+    request_changes_workflow: false
+    high_level_summary: true
+    poem: false
+    review_status: false
+    collapse_walkthrough: true
+    path_filters:
+        - "!api/"
+    path_instructions:
+        - path: "**/*.go"
+          instructions: "Review the Golang code for conformity with the Uber Golang style guide, highlighting any deviations."
+        - path: "tests/e2e/*"
+          instructions: |
+              "Assess the e2e test code assessing sufficient code coverage for the changes associated in the pull request"
+        - path: "tests/integration/*"
+          instructions: |
+              "Assess the e2e test code assessing sufficient code coverage for the changes associated in the pull request"
+        - path: "**/*_test.go"
+          instructions: |
+              "Assess the unit test code assessing sufficient code coverage for the changes associated in the pull request"
+        - path: "**/*.md"
+          instructions: |
+              "Assess the documentation for misspellings, grammatical errors, missing documentation and correctness"
+        - path: ".changelog/*"
+          instructions: |
+              "Assess the changes in the changelog for correctness and completeness, particularly flagging missing changes"
+    auto_review:
+        enabled: true
+        ignore_title_keywords:
+            - "WIP"
+            - "DO NOT MERGE"
+        drafts: false
```

Comment:
Does this mean it won't act on drafts before they are "ready for review"?
If yes, it might make sense to change this. If we had coderabbit comments already in the draft, we could apply its suggestions before publishing it for review.

Reply:
That's right. I don't know, I think it's sufficient to get comments when you put it out of draft; my fear is that it spams draft PRs.
afaict you can get it to review outside of the auto-review guidelines by commenting @coderabbit review
(see the tips that are collapsed at the bottom of its review)